### DISCUSSION TITLE
Roto League roster stats queried every day for further store into Excel or DB

### DISCUSSION THREAD

**Discussion by TheNighthawk99** [cwendt94/espn-api#405] [Q&A]
Hello,
I play ESPN Fantasy Basketball in a Roto-type public league. My goal is to gather all my roster player stats every day, then put them in an Excel file or DB (InfluxDB).
Currently, I do these action manually by writing down  my players stats every day by visiting the according pages at ESPN website.
Is this app suitable to accomplish that?
I don’t know python programming, I don’t know what is the starting point, so apologize me for this silly question.

**Comment by cwendt94** [comment]
Its definitely possible but you would need to write some python code to export the player stats to an Excel spreadsheet. With this package you can get your players stats for each game with the `player_info` functionality then take that data and write to a spreadsheet. Check out the [wiki](https://github.com/cwendt94/espn-api/wiki/League-Class-Basketball) for this package docs, and there are a lot of packages in python that can write to Excel. Here is a [link](https://www.geeksforgeeks.org/writing-excel-sheet-using-python/) I just looked up
